spec/steps/prepare.fmf

```diff
@@ -40,3 +40,33 @@ description: |
             playbooks:
                 - common.yml
                 - rhel7.yml
+
+/download:
+    summary:
+        Download artifact to the specificed directory on the target machine.
+    description:
+        Downloads given artifact to the specified destination directory.
+
+        Currently these artifacts are supported:
+            * brew or koji builds - can be specified by URL
+    example: |
+        prepare:
+            how: download
+            artifact: https://koji.fedoraproject.org/koji/buildinfo?buildID=1453726
+            destination: /tmp/systemd-244.1-3.fc32
+
+/download-multi:
```

Comment:
I think we can group both examples under a single story. Will be better for tracking and also generated online docs will be more clear.